language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -40,18 +54,32 @@ optionsParser = Options
         <> metavar "DIR"
         <> help "Output directory for the generated TypeScript files"
         )
+    <*> optional (strOption
+        (  long "daml-types-version"
+        <> help "The version of '@daml/types' to depend on (warning : not a user-configuration parameter; for SDK internal use)"
```

Comment:
If it’s internal add `<> internal` which hides it from `--help`.

Reply:
thanks